### PR TITLE
Entry and exit point for verse marker mode

### LINKED ISSUE #565: Add icon in the playback screen to enter "verse marker mode"

### PR DESCRIPTION
**Pull request by vleong2332**
This simply provides an entry point to and from the verse-marker mode on playback screen. In other words, showing and hiding buttons. No actual functionality has been written yet.

Should resolve #565

### CODE REVIEW

File: RecordingApp/app/src/main/java/wycliffeassociates/recordingapp/Playback/PlaybackScreen.java [hunk 1]
```diff
@@ -400,20 +419,76 @@ private void setButtonHandlers() {
...
+    }
+
+    private int getVerseCount() {
+        // NOTE: Replace with real code to get the number of verses in a chunk
+        int verses = 3;
+        // -1 because the first verse marker should be dropped at the beginning automatically
+        return verses - 1;
+    }
+
+    private void setVMarkerCount(int count) {
```
Comment (jsarabia1247) [inline RecordingApp/app/src/main/java/wycliffeassociates/recordingapp/Playback/PlaybackScreen.java]: Should we stick with a consistent naming convention here?
line 442 uses the full word Verse, here it is only V

I'd lean towards spelling out the word due to the need for plural in 438?

File: RecordingApp/app/src/main/java/wycliffeassociates/recordingapp/Playback/PlaybackScreen.java [hunk 2]
```diff
@@ -463,8 +538,27 @@ public void onClick(View v) {
...
+                    enterVMarkerMode();
+                    break;
+                }
+                case R.id.btnExitVMarkerMode: {
+                    exitVMarkerMode();
+                    break;
+                }
+                case R.id.btnDropVMarker: {
+                    dropVMarker();
+                    mVerses -= 1;
```
Comment (jsarabia1247) [inline RecordingApp/app/src/main/java/wycliffeassociates/recordingapp/Playback/PlaybackScreen.java]: should mVerses be renamed to mVersesToMark?
decrementing mVerses as markers are placed seems counter-intuitive

Comment (vleong2332) [inline RecordingApp/app/src/main/java/wycliffeassociates/recordingapp/Playback/PlaybackScreen.java]: Fair enough.

File: RecordingApp/app/src/main/res/layout/playback_screen.xml [hunk 3]
```diff
@@ -92,7 +126,7 @@
             android:gravity="center_vertical|right">
 
             <ImageButton
-                android:id="@+id/btnMarkerMode"
+                android:id="@+id/btnEnterVMarkerMode"
```
Comment (vleong2332) [inline RecordingApp/app/src/main/res/layout/playback_screen.xml]: Yes. This must have been changed before we decided.

**Pull request merged**